share/man/man7/mitigations.7

```diff
@@ -344,14 +346,24 @@ See
 for more information.
 .Pp
 .Sh HARDWARE VULNERABILITY MITIGATIONS
+.Ss Firmware and Microcode
 Recent years have seen an unending stream of new hardware vulnerabilities,
 notably CPU ones generally caused by detectable microarchitectural side-effects
 of speculative execution which leak private data from some other thread or
 process or sometimes even internal CPU state that is normally inaccessible.
 Hardware vendors usually address these vulnerabilities as they are discovered by
 releasing microcode updates, which may then be bundled into platform firmware
 updates
-.Pq historically called BIOS updates for PCs .
+.Pq historically called BIOS updates for PCs
+or packages to be updated by the operating system at runtime.
```

Comment:
Optional: I think we should probably mention that it's generally recommended to load the update as early as possible (e.g. via the Firmware Interface Table or early BIOS update, or so called BIOS updates) because it helps to provide better coverage for early boot stages, including the OS boot process itself (e.g. BIOS update better than loading microcode with `cpu_microcode_load` better than loading it at runtime).